unit-testing-react-what-why-how/Button/problem/.codesandbox/template.json

```diff
@@ -1,9 +0,0 @@
-{
-  "title": "Button",
-  "description": "Part of the “Unit Testing React: What, Why, How” training.",
-  "iconUrl": "https://www.bitovi.com/favicon.ico",
-  "tags": [
-      "html"
-  ],
-  "published": false
-}
```

Comment:
Was this deleted intentionally?

Reply:
this is just git being weird i think. copied over from another exercise and changed it.